src/PrestaShopBundle/Controller/Admin/Sell/Catalog/AttachmentController.php

```diff
@@ -356,9 +338,6 @@ private function getBulkAttachmentsFromRequest(Request $request): array
         return $attachmentIds;
     }
 
-    /**
-     * @return array
```

Comment:
while these comments look useless, we still should keep them as agreed with @matks